translator/trace/jaegerthrift_to_protospan.go

```diff
@@ -227,15 +228,12 @@ func jtagsToAttributes(tags []*jaeger.Tag) (string, tracepb.Span_SpanKind, *trac
 		case "http.status_code", "status.code": // It is expected to be an int
 			statusCodePtr = new(int32)
 			*statusCodePtr = int32(tag.GetVLong())
-			continue
```

Comment:
Why did you remove this? If we are capturing the status code as per the values declared in Lines 210 and 211 we don't need to add these as attributes.

Reply:
The issue is that the OC spec indicates that we should be actually mapping the values to the OC ones (see https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/HTTP.md#mapping-from-http-status-codes-to-trace-status-codes), with that it is not possible to restore the original keys from OC -> Jaeger, so dropping the key-value pairs is losing information that some systems in the backend may be relying upon.

At this stage it seems that a better general approach is: it is ok/desirable to lift information from known key-value pairs but we should preserve original pairs since in the backend there may be systems relying on these tags "http.status_code" seems a very likely one for this case.

Reply:
Oh okay I see, thanks for the references. As a minor point, in that case we need to then separate `http.status_code` and translate them to OpenCensus status codes.

Reply:
>  in that case we need to then separate http.status_code and translate them to OpenCensus status codes.

Agreed, created #238 to track it.